src/ingest/ingest_kba.py

```diff
@@ -58,7 +58,7 @@ def ingest_kba() -> None:
     create_id_index_if_not_exists(
         table_name="geometries_kba",
         index_name="idx_geometries_kba_sitrecid_text",
-        column="sitrecid::text",
+        column="sitrecid",
```

Comment:
@yellowcap Ah, sorry I ran the raw SQL locally and didn't test the script. We already create an index for the numeric value in the previous line - this one was intentionally created for text version of the id.

I think something like this will work for casting:

```suggestion
        column="(sitrecid::text)",
```